src/workerd/io/compatibility-date.capnp

```diff
@@ -444,7 +453,8 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
 
   nodeJsCompatV2 @50 :Bool
       $compatEnableFlag("nodejs_compat_v2")
-      $compatDisableFlag("no_nodejs_compat_v2");
+      $compatDisableFlag("no_nodejs_compat_v2")
+      $impliedByAfterDate(name = "nodeJsCompat", date = "2024-08-05");
```

Comment:
is this the date we want to draw in the sand for now? 

@irvinebroque how do you feel about this date? Could we push it out to September 1? I'm concerned that early august for on by default is a bit too aggressive. We should announce this and give it at least a month of opt-in usage before it's on by default.

We can announce in mid/late july, and make it the default in early September (way before birthday week to avoid collisions with other launches)

Reply:
See internal thread

Reply:
Date agreed on internally is 2024-09-02 ... PR is updated to reflect that.